src/ingestion/sources/base.py

```diff
@@ -132,8 +132,8 @@ def _execute(self, sql: str):
         return results
 
     def test(self):
```

Comment:
why change this file?

Reply:
because `raise e` wasn't doing what it was expected to do (propagate error to parent catch). Instead of raising an exception, it was raising the `e` message. This was breaking `self._initialize()` with a 500 that wasn't caught by any `Exception`. Instead, using just raise just propagates the error up to the parent.

Reply:
gross but okay